testing-chocolatey/local-testing.md

```diff
@@ -0,0 +1,77 @@
+# Beta-Testing Chocolatey with the Ansible Collection
+
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+  - [Vagrant](#vagrant)
+  - [Generic](#generic)
+- [Running `ansible-test`](#running-ansible-test)
+
+## Prerequisites
+
+Either:
+
+- [Vagrant][vagrant-download]
+- [VirtualBox][vbox-download]
+
+Or:
+
+- Windows VM
+- Linux VM (Ubuntu 18.10)
+
+[vagrant-download]: https://www.vagrantup.com/downloads
+[vbox-download]: https://www.virtualbox.org/wiki/Downloads
+
+## Getting Started
+
+### Vagrant
+
+1. Download the nupkg containing the Chocolatey version you'd like to test.
+1. Place the nupkg in the `provision_files` folder.
```

Comment:
I don't see `provision_files` as a folder in this repo. It _might_ be good to call out that if you clone this repo to use the vagrant file you'll need to create that folder.

Reply:
I added an empty `.gitkeep` file in this PR that will ensure that folder is created when the repo is cloned. 🙂

Reply:
Perfect.